lib/src/utils/serialization_utils.dart

```diff
@@ -0,0 +1,23 @@
+import 'package:ledger_cardano/src/utils/hex_utils.dart';
+import 'package:ledger_flutter/ledger_flutter.dart';
+
+class SerializationUtils {
+  static void writerSerializedPath(ByteDataWriter writer, List<int> path) {
+    writer.writeUint8(path.length);
+    for (var index in path) {
+      writer.writeUint32(index);
+    }
+  }
+
+  static void writeSerializedHex(ByteDataWriter writer, String hexString) {
+    var bytes = hex.decode(hexString);
+    writer.writeUint8(bytes.length);
+    writer.write(bytes);
+  }
+
+  static void serializeUint64(ByteDataWriter writer, int value) {
```

Comment:
Should rename to `writeSerializedUint64`

Reply:
done